packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -391,8 +322,8 @@ bool _isPolygonClockwise(List<gmaps.LatLng> path) {
   var direction = 0.0;
   for (var i = 0; i < path.length; i++) {
     direction = direction +
-        ((path[(i + 1) % path.length].lat - path[i].lat) *
-            (path[(i + 1) % path.length].lng + path[i].lng));
+        ((path[(i + 1) % path.length].lat! - path[i].lat!) *
+            (path[(i + 1) % path.length].lng! + path[i].lng!));
```

Comment:
Why are these safe?

Reply:
Added some docs with regards to how this is used. It is safe now to use `!` now, because of the LatLng transformation function not returning maybe nulls anymore, and what this is actually looking at.